src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -158,6 +168,21 @@ def launch_fluent(
         default is ``True``. You can stop and start the streaming of the
         Fluent transcript subsequently via the method calls, ``transcript.start()``
         and ``transcript.stop()`` on the session object.
+    ui : FluentUI or str, optional
+        UI option of Fluent. Options are either the values of the ``FluentUI`` enum
```

Comment:
```suggestion
        Fluent user interface mode. Options are either the values of the ``FluentUI`` enum
```
(same as other suggestion above)